site/docs/voice/errors.mdx

```diff
@@ -70,7 +70,7 @@ Content-Type: application/json;charset=UTF-8
 </TabItem>
 <TabItem value="403">
 
-Bandwidth returns a `HTTP-403` error when the user does not have access to the messaging API.
+Bandwidth returns a `HTTP-403` error when the user does not have access to the Voice API.
```

Comment:
A note for the weary PR reader: This is in the voice docs, so wrong API referenced.